pkg/api/describe/describer.go

```diff
@@ -97,7 +97,7 @@ func describeBuilderImage(config *api.Config, image string, out io.Writer) {
 		Tag:                config.Tag,
 		IncrementalAuthentication: config.IncrementalAuthentication,
 	}
-	build.GenerateConfigFromLabels(c)
+	build.GenerateConfigFromLabels(c, false)
```

Comment:
yes i took this as "describe as much as you can" .... I think we should leave this as is